src/scales/scale.radialLinear.js

```diff
@@ -348,7 +348,10 @@ module.exports = LinearScaleBase.extend({
 		LinearScaleBase.prototype.convertTicksToLabels.call(me);
 
 		// Point labels
-		me.pointLabels = me.chart.data.labels.map(me.options.pointLabels.callback, me);
+		me.pointLabels = me.chart.data.labels.map(function() {
+			var label = helpers.callback(me.options.pointLabels.callback, arguments, me);
+			return label || label === 0 ? label : '';
```

Comment:
Maybe `isNaN(label) ? '' : label;` or `helpers.isFinite(label) ? '' : label;` would be an easier way to write this?

Reply:
For backward compatibility, I would replace all falsy values except for `0` with `''`.

Reply:
@benmccann why should we only display numbers here?

Reply:
![image](https://user-images.githubusercontent.com/3874900/57859510-64002480-77f3-11e9-8d72-7b067f3b2d93.png)

Reply:
I didn't mean to suggest we should display only numbers. I saw only numbers given as examples and just didn't think about the text case since I'm not as familiar with the radar chart. This PR looks fine to me. I'm happy to merge as is